📝 Rework local tethered development (#2933)
platformsh/platformsh-docs: docs/layouts/shortcodes/local-dev/next-steps-end.md

> 1.  Commit and push the revisions by running the following command:

Reviewer: 🛠️ 

```suggestion
1.  To commit and push the revisions, run the following command:
```

    ```bash
    git add . && git commit -m "Add local configuration" && git push platform local-config
    ```

1.  Merge the change into production.

Once the script is merged into production,

> any user can then set up their local environment by running the following commands:

Reviewer: ⛏️  Seems redundant with "once".

```suggestion
any user can set up their local environment by running the following commands:
```
